docs/en/essentials/navigation.md

```diff
@@ -30,7 +30,7 @@ router.push({ name: 'user', params: { userId: 123 }})
 router.push({ path: 'register', query: { plan: 'private' }})
 ```
 
-**Note**: `params` are ignored if a `path` is provided, which is not the case for `query`, as shown in the example above. Instead, you need to provide the `name` of the route or manually specify the whole `path` with any parameter:
+**Note**: `params` are ignored if a `path` is provided, which is not the case for `query`, as shown in the example below. Instead, you need to provide the `name` of the route or manually specify the whole `path` with any parameter:
```

Comment:
As my understanding, the thing "... the case for `query`, as shown in the example ..." means `router.push({ path: 'register', query: { plan: 'private' }})` __above__, not __below__. 🤔
Thanks.

Reply:
ah, sorry, my mistaking ... 🙇

Reply:
no problem! 🙂